### ISSUE TITLE
Observation about ReLU activation function used

### ISSUE THREAD

**Issue by shashvatshah9** [mohammadpz/pytorch_forward_forward#15]
I just tried to implement the network training with GELU activation and got a reduced error for training and prediction for both layers.

```
train error: 0.06244003772735596
test error: 0.0626000165939331
```

I think adding a stochastic regularizer like dropout with layers would improve forward forward networks.
